packages/dashboard-frontend/src/pages/GetStarted/GetStartedTab/ImportFromGit/index.tsx

```diff
@@ -60,38 +57,9 @@ export class ImportFromGit extends React.PureComponent<Props, State> {
   }
 
   private async handleLocationChange(location: string): Promise<void> {
-    // if devWorkspace is enabled
-    // use factory workflow to load the git location
-    const cheDevworkspaceEnabled = isDevworkspacesEnabled(this.props.workspacesSettings);
-    if (cheDevworkspaceEnabled) {
-      const factoryUrl = `${window.location.origin}/#${location}`;
-      // open a new page to handle that
-      window.open(factoryUrl, '_blank');
-      return;
-    }
-
-    try {
-      this.setState({ isLoading: true });
-      await this.props.requestFactoryResolver(location);
-      // at this point the resolver object is defined
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      const resolver = this.factoryResolver.resolver!;
-      this.props.onDevfileResolve(resolver, location);
-      this.setState({ isLoading: false });
-    } catch (e) {
-      this.setState({ isLoading: false });
-      this.devfileLocationRef.current?.invalidateInput();
-      this.showAlert({
-        key: 'load-devfile-resolver-failed',
-        title: common.helpers.errors.getMessage(e),
-        variant: AlertVariant.danger,
-      });
-    }
-  }
-
-  private showAlert(alert: AlertItem): void {
```

Comment:
Then you need to get rid of the `removeAlerts` method and `alerts` field in the component state. Also, `factoryResolver` store is connected but not used in the component.

Reply:
I have fixed it.